full-service/src/json_rpc/address.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020-2021 MobileCoin Inc.
+
+use crate::db::AssignedSubaddress;
+use serde_derive::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize, Default, Debug, Clone)]
+pub struct Address {
+    pub object: String,
+    pub address_id: String,
+    pub public_address: String,
+    pub account_id: String,
+    pub address_book_entry_id: Option<String>,
+    pub comment: String,
+    pub subaddress_index: String,
+    pub offset_count: i32,
```

Comment:
Agreed!

Reply:
Note - these are currently documented in the [API.md](https://github.com/mobilecoinofficial/full-service/blob/main/API.md#the-assigned-address-object) - it would be nice to move that documentation in with the json_rpc object, and autogenerate something like API.md. API.md was modeled off of stripe's documentation.